src/extension/core/src/main/java/au/org/emii/core/Config.java

```diff
@@ -0,0 +1,153 @@
+package au.org.emii.core;
```

Comment:
If this is just the ncwms abomination config class then perhaps we should remove the other and reference this one rather than just duplicating the code.